compiler/x/amd64/codegen/AMD64SystemLinkage.cpp

```diff
@@ -724,6 +724,7 @@ TR::Register *
 TR_AMD64SystemLinkage::buildIndirectDispatch(TR::Node *callNode)
    {
    TR::SymbolReference *methodSymRef = callNode->getSymbolReference();
+   TR_ASSERT(methodSymRef->getSymbol()->castToMethodSymbol()->isComputedStatic(), "system linkage only supports computed static call for now %p\n", callNode);
```

Comment:
Should we provide this assert in the system linkage implementations for other platforms too?